packages/xo-server/src/api/vm.js

```diff
@@ -621,6 +621,9 @@ set.params = {
 
   // set the VM network interface controller
   nicType: { type: ['string', 'null'], optional: true },
+
+  // set the VM boot firmware mode
+  hvmBootFirmware: { type: 'string', optional: true },
```

Comment:
What value should be passed to set it to the default value?

Reply:
`null`

Reply:
Then `null` should be a valid type.